src/context/DatastoreReducer.js

```diff
@@ -35,9 +26,6 @@ export const DatastoreReducer = (state, action) => {
           workingList[item.measure] = item;
         }
       });
-      Object.keys(workingList).forEach((key) => {
-        workingList[key].label = createLabel(workingList[key].measure);
-      });
```

Comment:
So I actually do recommend that we append the labels provide from `info` to `currentResults`, which we use both in `MeasureResultsTable` and in the `D3IndicatorByLineSelector`. That way, we only need to run label matching once. Right now, the selector is broken under the "Measure By Line" chart:

![image](https://user-images.githubusercontent.com/4026454/158219796-eb819b0a-9f49-4568-a93c-c62948b69426.png)

Reply:
No actually that is a fair point. I changed it from `Promise.all` to individual promises because my trends were blowing up and partial data seemed better than no data. 

Hmmm. Huddle.